lib/client/conntest/connection_tester.go

```diff
@@ -18,12 +18,17 @@ package conntest
 
 import (
 	"context"
+	"time"
 
 	"github.com/gravitational/teleport/api/types"
 	"github.com/gravitational/teleport/lib/auth"
 	"github.com/gravitational/trace"
 )
 
+const (
+	DefaultDiagnosticDialTimeout = 5 * time.Second
```

Comment:
Nit: Move to lib/defaults as `DiagnosticDialTimeout` and add a godoc.

Also, maybe we already have an existing dial timeout constants with reasonable default and can just use it?